src/main/java/br/com/diegomelo/cleanparse/core/Parser.java

```diff
@@ -39,6 +38,11 @@ protected void validate(Object obj) throws ParserException {
 		List<String> fieldNames = new ArrayList<>();
 		Class<?> cls = obj.getClass();
 		Field[] fields = cls.getDeclaredFields();
+
+//test sonar
+Field test;
+
+Field test2;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "test2" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://35.202.0.216/coding_rules#rule_key=squid%3AS1481)